### ISSUE TITLE
TOC style of todo keyword is always 'OrgTodo'

### ISSUE THREAD

**Issue by hpgisler** [kjambunathan/org-mode-ox-odt#78]
The headers in the TOC style the todo-keywords worngly, if it is of type 'done'.
I would expect it to be 'OrgDone' if the keyword type is 'done'.

Note for the actual headers in the text (non-toc), this works as expected.

**Comment by kjambunathan** [comment]
> The headers in the TOC style the todo-keywords worngly, if it is of type 'done'.
> I would expect it to be 'OrgDone' if the keyword type is 'done'.
> 
> Note for the actual headers in the text (non-toc), this works as expected.

I have pushed a fix to the repo.    

(I have not updated the ELPA tar.  I will update it once I get couple of more fixes in.)

When I export this snippet,

```
#+options: ':nil *:t -:t ::t <:t H:3 \n:nil ^:t arch:headline
#+options: author:nil broken-links:mark c:nil creator:nil
#+options: d:(not "LOGBOOK") date:nil e:t email:t f:t inline:t num:t
#+options: p:nil pri:nil prop:nil stat:t tags:t tasks:t tex:t
#+options: timestamp:t title:t toc:t todo:t |:t

#+odt_preferred_output_format: pdf

#+odt_extra_styles: <style:style style:name="OrgTodo" style:family="text">
#+odt_extra_styles:  <style:text-properties fo:color="#ff0000"/>
#+odt_extra_styles: </style:style>
#+odt_extra_styles: <style:style style:name="OrgDone" style:family="text">
#+odt_extra_styles:  <style:text-properties fo:color="#18a303"/>
#+odt_extra_styles: </style:style>

* TODO A

* DONE B

```

I get this

![Screenshot from 2020-07-16 11-50-51](https://user-images.githubusercontent.com/19869864/87635000-f5eb2080-c75b-11ea-9e83-6c091fd19c35.png)

**Comment by kjambunathan** [comment]
This issue is fixed in `ox-odt-9.3.7.319.tar`
